src/main/java/com/github/fishio/control/MainMenuController.java

```diff
@@ -54,6 +58,7 @@ public void onSwitchTo() { }
 	 */
 	@FXML
 	public void startSinglePlayer() {
+		log.log(LogLevel.INFO, "Player Pressed the SingPlayer Button");
```

Comment:
Chiel, I think that if fails to load the image there, the program should still be able to keep running.

<naming